content/blog/methodo/importance-design-system-pour-UX-efficaces.md

```diff
@@ -0,0 +1,205 @@
+par exemple, l’application devra-t-elle répondre à des critères d’accessibilité ? Ils seront un référentiel pour 
+les designers et les développeurs, qui pourront y revenir pour s’assurer que les interfaces, 
+tant dans leur aspect visuel que dans leurs fonctionnalités répondent aux objectifs.
+- Déterminer la stratégie visuelle et l’image de marque, avec tous les éléments qu’elles comportent 
+(couleurs, typographies, éléments graphiques, illustrations, etc…), afin de concevoir le style guide. 
+Ce dernier sera l’un des premiers éléments à être intégrés dans le design system comme base 
+pour tous les composants qui seront créés par la suite.
+- Construire un kit UI, comme expliqué plus tôt, sur les principes d’atomic design 
+(c’est notamment là qu’intervient le style guide !) pour s’assurer qu’ils sont adaptables 
+et réutilisables dans plusieurs projets.
+
+Il est intéressant de noter que certains design sytems sont disponibles en open source, 
+tels que Material Design de Google, dont l’interface présente l’avantage d’être particulièrement bien adapté pour tout type d’appareil et toute résolution d’affichage. 
+Cet accès permet non seulement de pouvoir réutiliser des assets ou des patterns 
+et ainsi de bénéficier d’un gain de temps dans ses propres projets, 
+mais également d’en apprendre plus sur la construction des design systems en eux même. 
+
+### Les principes fondateurs d’un design system efficace 
+
+Plusieurs aspects essentiels peuvent être identifiés, en plus des éléments constitutifs du design system, qui, 
+une fois mis en œuvre, font de ces outils des ressources fiables. 
+Ces aspects favorisent considérablement la collaboration et simplifient la conception de produits multiplateformes. 
+Voici quelques notions :
+
+- Un processus itératif : un bon design system est évolutif, adaptable et réutilisable. 
+Il est construit de manière itérative, avec un accès aux anciennes versions ainsi qu’un historique 
+listant les modifications et apports à chaque nouvelle version.
+- Un article posté sur le site designsystems.com (Figma) à propos d’Encore, le design system de Spotify, 
+met en lumière l’importance de la flexibilité et de la prise en compte du cross-platform dans la création 
+et la mise à jour de design system. Ces deux principes permettent de créer une vraie cohérence 
+entre les interfaces selon les plateformes utilisées (web ou mobile) et les tailles d’écran, 
+résultant ainsi en une expérience utilisateur plus fluide et intuitive.
+- L’accessibilité comme l’un des principes fondateurs : Toujours sur le site designsystems.com, 
+Figma a réalisé une série d’articles sur le futur des design systems. Ils sont amenés à se complexifier en même temps 
+que les projets et devront non seulement être adaptés (notamment grâce à l’automatisation par l’IA) 
+mais ils seront également  tenus de prendre en compte et d’intégrer les principes d’accessibilité, 
```

Comment:
Ajouter ce lien sur "les principes d'accessibilité" https://www.designsystems.com/the-future-of-design-systems-is-accessible/